books/RayTracingTheRestOfYourLife.html

```diff
@@ -2377,47 +2381,6 @@
     [Listing [emitted-directional]: <kbd>[material.h]</kbd> Material emission, directional]
 </div>
 
-<div class='together'>
-We also need to flip the light so its normals point in the $-y$ direction:
-
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
```

Comment:
Why was this section removed? Are lights now double sided?

Reply:
We had to flip XZ rects, because they always point up. With quads, you can specify them in either orientation.

Reply:
Should we maybe call out the fact that quad lights are one sided?

Reply:
Ehhh, no need. We already discuss sidedness in chapter 6 of book 1.